src/backend/options.ml

```diff
@@ -9,10 +9,11 @@ open Cmdliner
 type t = {
   db_path : string;
   server_port : int;
+  static_path : string;
```

Comment:
As stated in another comment, we currently "crunch" all of the static files so that the `ftw` binary is standalone, so this should not be needed.